_pages/material.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "Apache Beam Material"
+permalink: /material/
+---
+
+This page contains project material for the Apache Beam project.
+
+## Project logos
+You can download [this archive]({{ site.baseurl }}/{{ site.downloads }}/{{ site.data.logos.archive-file }})
```

Comment:
Nit: There seems to be unnecessary files in the archive. `__MACOSX`, various `.DS_Store` files. Can we clean them up?